test/integration/data-types.test.js

```diff
@@ -175,9 +175,12 @@ describe(Support.getTestDialectTeaser('DataTypes'), () => {
     return testSuccess(Type, 'foobar');
   });
 
-  it('calls parse and stringify for STRING', () => {
+  it('calls parse and stringify/bindParam for STRING', () => {
     const Type = new Sequelize.STRING();
 
+    if (dialect === 'mssql') {
```

Comment:
Why only mssql?

Reply:
mssql has a `_bindParam` function that outputs either a Buffer or a normal string dependent on whether or not the `STRING` data type uses the binary param. This override isn't needed for other dialects - I've added a comment.